tools/shared/pragma.c

```diff
@@ -271,6 +271,63 @@ static void lcase(char *);
 static bool craydir; /* true if cray directive */
 static bool sundir;  /* true if sun directive */
 
+/*
+ * reads vectorlength directive () set proper flags or raise error)
+ */
+static void scan_vectorlength(int* toSet,int* Num, int typ)
+{
+  int backup_nowarn = gbl.nowarn;
+  if (gtok() != T_LP){
```

Comment:
Add space between ) and {, namely `) {`

Reply:
I would like to see this tidied up as well.